dlme_airflow/task_groups/etl.py

```diff
@@ -19,13 +19,12 @@
 def etl_tasks(provider, task_group: TaskGroup, dag: DAG) -> TaskGroup:
     task_array = []
     source = catalog_for_provider(provider)
-    logging.info(f'The source is {source}')
     try:
         collections = iter(list(source))
         for collection in collections:
-            task_array.append(build_collection_etl_taskgroup(provider, collection, "/opt/dlme_airflow/utils/ifpo_get_thumbnail_urls.py", task_group, dag))
+            task_array.append(build_collection_etl_taskgroup(provider, collection, task_group, dag))
```

Comment:
This was too early to try to include the post_harvest config, we will fetch it further down the stack.